### DISCUSSION TITLE
日本語で話しましょう！

### DISCUSSION THREAD

**Discussion by SethClydesdale** [SethClydesdale/genki-study-resources#82] [Discussions]
始めましょう！

私の好きな食べ物はタコスとピザですよ。🤤 

あなたの好きな食べ物は何ですか。

**Comment by SethClydesdale** [comment]
昨日、遂に閃の軌跡IIを始めました。八ヶ月前、閃の軌跡Iを終えました。それから、空の軌跡と零／碧の軌跡と那由多の軌跡をしました。とても楽しかったですよ。

**Comment by SethClydesdale** [comment]
最近はファンゲームを作っています。日本語と英語のタイピング練習ゲームですよ。もう少しで完成します。

ドラフト動画：https://twitter.com/SethC1995/status/1302342732787134470

**Reply by SethClydesdale** [reply]
遂に完成しました…
https://sethclydesdale.github.io/typing-with-mishy/

**Comment by chihiro888** [comment]
めちゃ素敵なプロジェクトですよね！
モチベーションになりました！

**Reply by SethClydesdale** [reply]
ありがとう、嬉しいです。😊

**Comment by SethClydesdale** [comment]
誕生日の休みは終わりました。
頑張ります！

**Comment by SethClydesdale** [comment]
ごめんなさい、最近はモチベーションが…ありません。m(__)m
とにかく来週は頑張ってみます。

**Comment by SethClydesdale** [comment]
メリークリスマス！

**Comment by SethClydesdale** [comment]
あけましておめでとうございます！
